statsmodels/stats/dist_dependence_measures.py

```diff
@@ -0,0 +1,505 @@
+# -*- coding: utf-8 -*-
+""" Distance dependence measure and the dCov test.
+
+Implementation of Székely et al. (2007) calculation of distance
+dependence statistics, including the Distance covariance (dCov) test
+for independence of random vectors of arbitrary length.
+
+Author: Ron Itzikovitch
+
+References
+----------
+.. Szekely, G.J., Rizzo, M.L., and Bakirov, N.K. (2007)
+   "Measuring and testing dependence by correlation of distances".
+   Annals of Statistics, Vol. 35 No. 6, pp. 2769-2794.
+
+"""
+import numpy as np
+import warnings
+from collections import namedtuple
+
+from scipy.spatial.distance import pdist, squareform
+from scipy.stats import norm
+
+
+DistDependStat = namedtuple(
+    "DistDependStat",
+    ["test_statistic", "distance_correlation",
+     "distance_covariance", "dvar_x", "dvar_y", "S"],
+)
+
+
+def dcovtest(x, y, B=None, method="auto"):
```

Comment:
I missed one, sorry.  Probably distance_covariance_test, or perhaps test_distance_covariance.  Your choice. Then ready after green.

Reply:
missed it too :)
fixed